pkg/redis/providerresult.ipldsch

```diff
@@ -0,0 +1,18 @@
+type PeerID bytes
+type Multiaddr bytes
```

Comment:
```suggestion
type PeerID Bytes
type Multiaddr Bytes
```

Reply:
that actually didn't work. I think when you define alias types you have to use the lower case.